pkg/action/oci_test.go

```diff
@@ -44,14 +45,21 @@ func TestOCI(t *testing.T) {
 		t.Fatalf("full: %v", err)
 	}
 
+	sort.Slice(out.Bytes(), func(i, j int) bool {
+		return out.Bytes()[i] < out.Bytes()[j]
+	})
 	got := out.String()
 
 	td, err := os.ReadFile("testdata/scan_oci")
 	if err != nil {
 		t.Fatalf("testdata read failed: %v", err)
 	}
 	// Sort the loaded contents to ensure consistent ordering
+	sort.Slice(td, func(i, j int) bool {
+		return td[i] < td[j]
+	})
 	want := string(td)
+
```

Comment:
Same comment here - I think simple is the wrong output format for validating that we can scan OCI files.

Reply:
Updated in [`f2d2599` (#617)](https://github.com/chainguard-dev/malcontent/pull/617/commits/f2d2599b9505783c903b658b6b396c1c9b19b786).